code-examples/core_features/web3names/2_web3name.ts

```diff
@@ -0,0 +1,27 @@
+import { ApiPromise } from '@polkadot/api'
+import { KeyringPair } from '@kiltprotocol/types'
+
+import { DemoKeystore, FullDidDetails, Web3Names } from '@kiltprotocol/did'
+import { SubscriptionPromise } from '@kiltprotocol/types'
+import { BlockchainUtils } from '@kiltprotocol/chain-helpers'
+
+export async function main(
+  api: ApiPromise,
+  keystore: DemoKeystore,
+  kiltAccount: KeyringPair,
+  didOwner: FullDidDetails,
+  web3Name: Web3Names.Web3Name,
+  resolveOn: SubscriptionPromise.ResultEvaluator = BlockchainUtils.IS_FINALIZED
+) {
+  // Release the Web3 name if the owner is the provided DID.
+  const web3NameOwner = await Web3Names.queryDidForWeb3Name(web3Name)
+  if (web3NameOwner === didOwner.did) {
+    const web3NameReleaseTx = await Web3Names.getReleaseByOwnerTx().then((tx) =>
+      didOwner.authorizeExtrinsic(tx, keystore, kiltAccount.address)
+    )
+    await BlockchainUtils.signAndSubmitTx(web3NameReleaseTx, kiltAccount, {
+      reSign: true,
+      resolveOn
+    })
+  }
```

Comment:
Same argument for conciseness. We also don't check the deposit owner in [3_web3name.ts](https://github.com/KILTprotocol/docs/pull/62/files#diff-393984dde483c5772926aa88635b1fe69da0fb2236619b1bfd6ace6178702dfc)

```suggestion
  // Release the Web3 name. This will fail if `web3Name` is not owned by `didOwner`.
  const web3NameReleaseTx = await Web3Names.getReleaseByOwnerTx().then((tx) =>
    didOwner.authorizeExtrinsic(tx, keystore, kiltAccount.address)
  )
  await BlockchainUtils.signAndSubmitTx(web3NameReleaseTx, kiltAccount, {
    reSign: true,
    resolveOn
  })
```

Reply:
https://github.com/KILTprotocol/docs/pull/62/commits/dc9c6f81948c41c29dbc56979e45e2c8bcf9d300